force-app/main/default/classes/ADDR_Addresses_UTIL_TEST.cls

```diff
@@ -0,0 +1,1341 @@
+        address.MailingStreet__c = 'test\r\nmultiple line street';
+
+        Test.startTest();
+        ADDR_Addresses_UTIL.handleMultilineStreet(address);
+        Test.stopTest();
+
+        System.assertEquals('test', address.MailingStreet__c);
+        System.assertEquals('multiple line street', address.MailingStreet2__c);
+    }
+
+    @isTest
+    static void testHandleMultilineStreetN() {
+        Address__c address = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1)[0];
+        address.MailingStreet__c = 'test\nmultiple line street';
+
+        Test.startTest();
+        ADDR_Addresses_UTIL.handleMultilineStreet(address);
+        Test.stopTest();
+
+        System.assertEquals('test', address.MailingStreet__c);
+        System.assertEquals('multiple line street', address.MailingStreet2__c);
+    }
+
+    /**************************************************************************************************************************
+     ****************************************************** getMultilineStreet ************************************************
+     **************************************************************************************************************************/
+    @isTest
+    static void testGetMultilineStreetNull() {
+        //To Refactor: we don't have guard clause for this.
+        /*Test.startTest();
+        ADDR_Addresses_UTIL.getMultilineStreet(null);
+        Test.stopTest();*/
+
+        //Expecting null pointer
+    }
+
+    @isTest
+    static void testGetMultilineStreetStreet2NotBlank() {
+        Address__c address = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1)[0];
+        address.MailingStreet__c = 'test';
+        address.MailingStreet2__c = 'multiple line street';
+        Test.startTest();
+        String multilineStreet = ADDR_Addresses_UTIL.getMultilineStreet(address);
+        Test.stopTest();
+
+        System.assertEquals('test\r\nmultiple line street', multilineStreet);
+    }
+
+    @isTest
+    static void testGetMultilineStreetStreet2Blank() {
+        Address__c address = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1)[0];
+        address.MailingStreet__c = 'test';
+        address.MailingStreet2__c = null;
+        Test.startTest();
+        String multilineStreet = ADDR_Addresses_UTIL.getMultilineStreet(address);
+        Test.stopTest();
+
+        System.assertEquals('test', multilineStreet);
+    }
+
+    /**************************************************************************************************************************
+     ****************************************************** isStateCountryPicklistsEnabled ************************************
+     **************************************************************************************************************************/
+
+    //At this moment, it is really hard to test this method. We may consider make the methods virtual to override.
```

Comment:
Write a test for what we can know. Don't be afraid to make multiple tests that will exit early if the org state isn't what we need.

Reply:
I tend to ignore the state picklist function at this moment since we won't touch them. Not ideal but refactoring it feels like a heavier lift and riskier.